builtin/logical/transit/path_keys.go

```diff
@@ -62,6 +62,13 @@ you ensure that all nonces are unique for a
 given context. Failing to do so will severely
 impact the ciphertext's security.`,
 			},
+
+			"exportable": &framework.FieldSchema{
+				Type: framework.TypeBool,
+				Description: `Enables keys to be exportable.
+This allows for all the valid keys
+in the key ring to be exported.`,
```

Comment:
I was going to say that this misses the upsert case, but I actually think until someone complains we should not support marking keys as exportable during the upsert case; once we get controls in for per-parameter ACLs it will be safer to do so.